### PR TITLE
[bitnami/airflow] Fix tolerations for Airflow Exporter

### PR DESCRIPTION
**Pull request by sgrzemski-ias**
**Description of the change**
Make Airflow Exporter respect the tolerations settings. Currently it is declared in values but it was not implemented in the deployment template for airflow exporter. They are also present in the docs, but not used -.-

**Benefits**
Airflow Exporter respect the tolerations settings and end users can use it in their setups. 

**Possible drawbacks**
None.

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/airflow/templates/metrics/deployment.yaml [hunk 1]
```diff
@@ -39,6 +39,13 @@ spec:
       nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.nodeSelector "context" $) | nindent 8 }}
       {{- end }}
       {{- end }}
+      {{- if or .Values.tolerations .Values.metrics.tolerations }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- else }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- end }}
```
Comment (alemorcuq) [inline bitnami/airflow/templates/metrics/deployment.yaml:48]: According to the `values.yaml`, the `metrics.tolerations` parameter takes precedence over the global tolerations parameter.

> Note: this configuration supersedes the global tolerations configuration

So this logic should be the other way around: render `metrics.tolerations` if they are set, default to `tolerations` instead.

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR! See my comment below.

File: bitnami/airflow/templates/metrics/deployment.yaml (hunk 1 above)
Comment (sgrzemski-ias) [reply bitnami/airflow/templates/metrics/deployment.yaml:48]: Yes, I totally agree with that, but I used the same strategy as is used with nodeselector. Let me fix that then.

Comment (sgrzemski-ias) [reply bitnami/airflow/templates/metrics/deployment.yaml]: Done. `metrics.tolerations` would be set if defined, if not `tolerations` would be set if defined.

**Review by alemorcuq — APPROVED** [review]
Thanks!

**Pull request merged**